packages/api/db/migration/20220903141736_alter_wage_object.js

```diff
@@ -0,0 +1,31 @@
+export const up = function (knex) {
+  return Promise.all([
+    knex.schema.alterTable('userFarm', (t) => {
+      t.jsonb('wage')
+        .defaultTo(
+          JSON.stringify({
+            type: 'hourly',
+            amount: 0,
+            never_ask: false,
+          }),
+        )
+        .alter();
+    }),
+  ]);
+};
+
+export const down = function (knex) {
+  return Promise.all([
```

Comment:
typically if I am adding a column in the up function then the down function should drop the same column